llvm/test/CodeGen/AArch64/fixed-point-conv-vec-pat.ll

```diff
@@ -0,0 +1,103 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 5
+; RUN: llc < %s | FileCheck %s
+
+target triple = "aarch64"
```

Comment:
Personally I think RUN lines should be kept to a minimum. I'd love a world where  `llc < file` always just worked, although appreciate this is wishful thinking because it's not always practical.